### PR TITLE
Add pauseTest command

### PR DESCRIPTION
**Pull request by jasonLaster**
Simple way to pause and then resume a test which lets you interact with the debugger.

### CODE REVIEW

File: docs/mochitests.md [hunk 1]
```diff
@@ -78,6 +78,19 @@ Any of the below APIs that takes a `url` will match it as a substring, meaning t
 * `waitForThreadEvents(dbg, eventName)` - Waits for specific thread events
 * `waitForDispatch(dbg, type)` - Wait for a specific action type to be dispatch. If an async action, will wait for it to be done.
 
+## Pausing
```
Comment (jlongster) [inline docs/mochitests.md]: Do we need a whole section for this? Why not just document the `pauseTest` helper in the API section? Keeping the docs smaller makes it easier to go through, although it's balance.

Comment (jasonLaster) [inline docs/mochitests.md]: That's a good point - happy to add it to the API and also add some of the newer methods like clickElement, findelement...

Comment (jlongster) [inline docs/mochitests.md]: You could rename this to `Debugging`, it probably is good to have a small section calling that out, and mentioning the `--jsdebugger` allows you to use the debugger and mention this command. Not sure we need to detail breakpoints/debugger specifically though :)

Thanks for this new function!

**Comment by jlongster** [comment]
👍 Looks good to me

File: docs/mochitests.md (hunk 1 above)
Comment (jasonLaster) [inline docs/mochitests.md]: thought about it a bit more and decided adding the command to the api section was fine

**Pull request merged**